bower.json

```diff
@@ -20,7 +20,8 @@
   "dependencies": {
     "polymer": "Polymer/polymer#v0.9.0-rc.1",
     "paper-dialog-behavior": "PolymerElements/paper-dialog-behavior#^0.9.0",
-    "paper-styles": "PolymerElements/paper-styles#^0.9.0"
+    "paper-styles": "PolymerElements/paper-styles#^0.9.0",
+    "neon-animation": "PolymerElements/neon-animation#^0.0.1"
```

Comment:
Also, what do you think about making the release line up with the other element catalogues? e.g., 0.9.x for Polymer@0.9.x.